src/components/login-page/LoginPage.jsx

```diff
@@ -0,0 +1,199 @@
+import CssBaseline from '@mui/material/CssBaseline';
+import Box from '@mui/material/Box';
+import { Button } from '@mui/material';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import { useState } from 'react';
+// import { useNavigate } from 'react-router-dom';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Stack from '@mui/material/Stack';
+import { grey } from '@mui/material/colors';
+import { useAuth } from '../../utils/constants';
+import CircularProgress from '@mui/material/CircularProgress';
+import Snackbar from '@mui/material/Snackbar';
+
+export function LoginPage() {
+    // const navigate = useNavigate();
+
+    const inputsStyle = {
+        marginBottom: '21px',
+    };
+    const labelsStyle = {
+        marginBottom: '7px',
+    };
+    const mySecondaryColor = grey[500];
+
+    const [userInput, setUserInput] = useState({
+        username: '',
+        password: '',
+    });
+
+    const [isLoading, setIsLoading] = useState(false);
+    const [isDisabled, setIsDisabled] = useState(false);
+    const [alertMsg, setAlertMsg] = useState({

```

Comment:
since the alertMsg state only changes the "open" part of the object, we can simply pass the "vertical" and "horizontal" values to  the SnackBar component as "top" and "center". That way we can make the state a bit more simple